docs/gettingstarted.rst

```diff
@@ -162,42 +166,82 @@ Run::
    better to wipe the database and start over.
 
 
-General architecture
-====================
-
-.. image:: block-diagram.png
-
-Arrows direction represents the flow of interesting information (crashes,
-authentication assertions, cached values), not trivia like acks.
-
+.. _gettingstarted-chapter-configuration:
 
 Configuration
--------------
+=============
 
 All configuration is done with ENV files located in ``/app/docker/config/``.
 
-Each service uses ``docker_common.env`` and then a service-specific ENV file.
-
 ``docker_common.env``
-    This holds secrets and environment-specific configuration required
+    This holds secrets and *environment-specific configuration* required
     to get services to work in a docker environment for local development.
 
     This should NOT be used for server environments.
 
+``local_development.env``
+    This holds a few environment variables that are explicitly for
+    local development and should never show up in a server environment.
+
+``processor.env``, ``crontabber.env``, and ``webapp.env``
+    These configuration files hold *behavioral configuration* for these three
+    components for all environments--local development and servers.
+
+    For example, if you want to add a new destination crash store to the system,
+    you'd add it to ``processor.env``.
+
 ``test.env``
     This holds configuration specific to running the tests. It has some
     configuration value overrides because the tests are "interesting".
 
-``processor.env`` and ``webapp.env``
-    These configuration files hold behavioral configuration for these two things
-    that work across ALL environments--local development and servers.
+``my.env``
+    This file lets you override any environment variables for your local
+    development environment.
 
-    For example, if you want to add a new destination crash store to the system,
-    you'd add it to ``processor.env``.
+    The template for this is in ``docker/config/my.env.dist``.
```

Comment:
I'd love an extra comment about something like "This is your personal file. It's never checked into git.".
Perhaps I'm wrong. Perhaps is "obvious" from the name.